tests/test_edgeql_data_migration.py

```diff
@@ -3808,15 +3872,31 @@ async def test_edgeql_migration_eq_33(self):
             }],
         )
 
-        await self.migrate(r"""
-            type Child;
-            type Child2;
+        await self.migrate(
+            r"""
+                type Child;
+                type Child2;
 
-            type Base {
-                # change link type
-                link foo -> Child2;
-            }
-        """)
+                type Base {
+                    link foo -> Child;
+                }
+            """,
+        )
+
+        await self.migrate(
+            r"""
+                type Child;
+                type Child2;
+
+                type Base {
+                    # change link type
+                    link foo -> Child2;
+                }
+            """,
+            user_input=[
```

Comment:
Could you add a test that requires two user inputs?

Reply:
Good point. Added that to `test_edgeql_migration_eq_collections_07`